5.1 Error - Cannot find module 'webpack-log' at run time

With the 5.1.0 release, the following error is occurring when trying to run the loader: 

```
Module build failed: Error: Cannot find module 'webpack-log'
    at Function.Module._resolveFilename (module.js:555:15)
```

It looks like `webpack-log` should have been added as a dependency instead of a dev-dependency.

--- a/package.json
+++ b/package.json
@@ -39,7 +39,8 @@
     "lodash": "^4.17.5",
     "micromatch": "^3.1.9",
     "mkdirp": "^0.5.1",
-    "source-map-support": "^0.5.3"
+    "source-map-support": "^0.5.3",
+    "webpack-log": "^1.2.0"
   },
   "peerDependencies": {
     "typescript": "^2.7"
@@ -69,7 +70,6 @@
     "tslint": "^5.10.0",
     "typescript": "^2.9.1",
     "webpack": "^4.12.0",
-    "webpack-cli": "^3.0.3",
-    "webpack-log": "^1.2.0"
+    "webpack-cli": "^3.0.3"
   }
 }

